tuist/tuist
# Add flag to opt out from beautifying `xcodebuild` logs

### Short description 📝
Tuist defaults to beautifying the xcodebuild logs using [xcbeautify](https://github.com/tuist/xcbeautify) without giving the users the option to opt-out from that. I'm adding a new flag, `--raw-xcodebuild-logs` to give users control, and I'm adjusting our acceptance tests to use it for the inner `xcodebuild` processes. As described [here](https://github.com/tuist/xcbeautify/issues/165), that results in xcodebuild logs beautified and mixed together, which complicates the debugging a bit, but hopefully that's something we'll be able to address soon once we tackle that PR. After this change is merge, we should be able to debug our acceptance tests in production using logs.

### How to test the changes locally 🧐
You can run a command like `build` against Tuist passing the `--raw-xcodebuild-logs` flag and you should see the raw `xcodebuild` logs.

### Contributor checklist ✅


### Reviewer checklist ✅
